### PR TITLE
chore: small script fixes and goerli deployment

### PR DESCRIPTION
**Pull request by kupermind**
- Goerli deployment.

### CODE REVIEW

**Review by 77ph — APPROVED** [review]

File: scripts/deployment/deploy_18_change_managers.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ async function main() {
 
     // Transaction signing and execution
     // 18. EOA to change the manager of ServiceRegistryTokenUtility to ServiceManagerToken calling `changeManager(ServiceManagerToken)`;
-    console.log("You are signing the following transaction: componentRegistry.connect(EOA).changeManager()");
+    console.log("You are signing the following transaction: serviceRegistryTokenUtility.connect(EOA).changeManager()");
```
Comment (mariapiamo) [inline scripts/deployment/deploy_18_change_managers.js:35]:
```suggestion
    console.log("You are signing the following transaction: serviceManagerTokenAddress.connect(EOA).changeManager()");
```

Comment (mariapiamo) [reply scripts/deployment/deploy_18_change_managers.js:35]: same below please

Comment (kupermind) [reply scripts/deployment/deploy_18_change_managers.js:35]: that's not really the case, but I understood what you meant

Comment (mariapiamo) [reply scripts/deployment/deploy_18_change_managers.js]: yes, sorry. You are right! Lgt

**Pull request merged**